### PR TITLE
add owner criteria to right

### PR DESCRIPTION
**Pull request by veolys-nanne**

### CODE REVIEW

File: WorkflowFunctionAdminBundle/Resources/views/Form/form_div_layout.html.twig [hunk 1]
```diff
@@ -19,7 +20,10 @@
                             {{ child.vars.label|trans({}, translation_domain)|raw }}
                             {{ form_widget(child.referenceId) }}
                         </td>
-                        {{ form_widget(child) }}
+                        {{ form_widget(child.workflowFunctions) }}
+                        <td>
+                        {{ form_widget(child.owner) }}
```
Comment (nicolasThal) [inline WorkflowFunctionAdminBundle/Resources/views/Form/form_div_layout.html.twig]: can you fix the indentation?

**Comment by nicolasThal** [comment]
Nice for the tests, you could have added a case with a mock of statusableInterface only to be complete

**Comment by veolys-nanne** [comment]
done @nicolasThal

File: WorkflowFunctionAdminBundle/Tests/Security/Authorization/Voter/WorkflowRightVoterTest.php [hunk 2]
```diff
@@ -138,11 +141,20 @@ public function provideObject()
...
 
         $object2 = Phake::mock('OpenOrchestra\ModelInterface\Model\NodeInterface');
+        Phake::when($object2)->getCreatedBy()->thenReturn($this->username);
         $object3 = Phake::mock('OpenOrchestra\ModelInterface\Model\NodeInterface');
+        Phake::when($object3)->getCreatedBy()->thenReturn($this->username);
+        $object4 = Phake::mock('OpenOrchestra\ModelInterface\Model\NodeInterface');
+        Phake::when($object4)->getCreatedBy()->thenReturn('fakeOtherUsername');
+
+        $object5 = Phake::mock('OpenOrchestra\ModelInterface\Model\StatusableInterface');
+        Phake::when($object5)->getCreatedBy()->thenReturn($this->username);
```
Comment (nicolasThal) [inline WorkflowFunctionAdminBundle/Tests/Security/Authorization/Voter/WorkflowRightVoterTest.php]: as the `$object5` is a mock of `StatusableInterface` the method `getCreatedBy` does not exists

**Comment by nicolasThal** [comment]
:+1:

**Pull request merged**